src/OrchardCore/OrchardCore.Infrastructure.Abstractions/Security/RoleClaim.cs

```diff
@@ -17,11 +18,32 @@ public class RoleClaim
     /// </summary>
     public string ClaimValue { get; set; }
 
+    public RoleClaim()
+    {
+    }
+
+    public RoleClaim(string value)
+    {
+        ClaimType = Permission.ClaimType;
+        ClaimValue = value;
+    }
+
+    public RoleClaim(string value, string type)
```

Comment:
Reason I like it how it is to have a constructor with no type since we use Permissions type by default. This was we pass value as first parameter. I don't want to have value as first parameter and then another constructor where type is the first parameter. I think it would be very confusing

Reply:
Agreed.

Reply:
@gvkries I cleaned up the constructors and now type is the first parameter. Let me know if you are good with the last commit.